### DISCUSSION TITLE
Request for RPM officially on Fedora

### DISCUSSION THREAD

**Discussion by Espionage724** [nokyan/resources#365] [Ideas]
I was reading GNOME 47 changes, saw the Circle section, saw Resources, looked into it, thought it looked pretty cool and wanted to try it out, but have been avoiding Flatpaks for a while (I prefer RPMs, binaries, or AppImage last-resort). Checked for `gnome-resources` in `dnf search` and didn't see anything (F41 beta).

Basically Resources looks like a cool app, but I would like to see it officially-provided on Fedora as a rpm I can install through `dnf`.

**Comment by nokyan** [comment]
Hi,
thanks for the feedback. There is an ___unoffical___ Copr repo for Resources, you can find it in the README.
Flatpak will remain the primary mean of distributing Resources simply because it allows me to move faster to e.g. new GNOME runtimes and especially because it spares me from having to deal with distro packaging shenanigans.
